homework/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
On a wide screen you UI could be improved by reducing the width and centering the content. Stretching it out across the screen is a bit over the top.

![image](https://user-images.githubusercontent.com/2788771/61948150-5a3f2e00-afa7-11e9-90c1-8744abb15019.png)